packages/test/test-utils/package.json

```diff
@@ -117,6 +117,10 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"ClassDeclaration_TestFluidObject": {
```

Comment:
Is it intended that test-utils is participating in type testing?

Reply:
If it's in the fluidframework scope, yeah. Default is to type test everything except private/unpublished packages or those that are explicitly opted out.

Reply:
i think this needs to go in next. partners do consume our test packages. i wish they didn't or that it was more structured, but that's the way it currently is.

Reply:
Already targeting next 👍